web/tests/lynxkite.ts

```diff
@@ -450,14 +430,14 @@ class PlotState extends PopupBase {
     for (let i = 0; i < count; ++i) {
       const path = await bars.nth(i).getAttribute('d')!;
```

Comment:
> Why didn't this exclamation mark work? Anyway, let's delete it since we have one below.

The `!` applied to the Promise returned by `getAttribute()` not to the string we `await`. Good catch! I've removed this.